<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Pclwindow</class>
 <widget class="QMainWindow" name="Pclwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>712</width>
    <height>570</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pclwindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="7" column="0">
     <widget class="Line" name="line_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QVTKWidget" name="widget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>601</width>
        <height>271</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <layout class="QGridLayout" name="buttonsLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item row="1" column="5">
       <widget class="QPushButton" name="yRIncButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>+ y</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QPushButton" name="xRIncButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>+ x</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="yTIncButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>+ y</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="zTIncButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>+ z</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="zTDecButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>- z</string>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QPushButton" name="xRDecButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>- x</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QLabel" name="rotationStepLabel">
        <property name="text">
         <string>step (rad)</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="6">
       <widget class="QPushButton" name="zRDecButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>- z</string>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <widget class="QLineEdit" name="rotationStepInput">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="translationStepInput">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="yTDecButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>- y</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="translationStepLabel">
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="text">
         <string>step (m)</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="RotationLabel">
        <property name="text">
         <string>Rotation</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="translationLabel">
        <property name="text">
         <string>Translation</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="xTIncButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>+ x</string>
        </property>
       </widget>
      </item>
      <item row="1" column="6">
       <widget class="QPushButton" name="zRIncButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>+ z</string>
        </property>
       </widget>
      </item>
      <item row="2" column="5">
       <widget class="QPushButton" name="yRDecButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>- y</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="xTDecButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>- x</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Edit controls:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <layout class="QGridLayout" name="gridLayout_2" rowstretch="0,0" rowminimumheight="0,0">
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="cloudSelectionLayout">
        <property name="leftMargin">
         <number>5</number>
        </property>
        <property name="topMargin">
         <number>5</number>
        </property>
        <property name="rightMargin">
         <number>5</number>
        </property>
        <property name="bottomMargin">
         <number>5</number>
        </property>
        <item>
         <widget class="QRadioButton" name="cloud0Radio">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>camera 0</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="cloud1Radio">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>camera 1</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="cloud2Radio">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>camera 2</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="cloud3Radio">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>camera 3</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Select the camera's cloud to edit:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Bounding Box:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="computeBBButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Compute</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>712</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="openFile0Action"/>
    <addaction name="openFile1Action"/>
    <addaction name="openFile2Action"/>
    <addaction name="openFile3Action"/>
    <addaction name="separator"/>
    <addaction name="exitAction"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="exitAction">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="openFile0Action">
   <property name="text">
    <string>Open cam-0 file</string>
   </property>
  </action>
  <action name="openFile1Action">
   <property name="text">
    <string>Open cam-1 file</string>
   </property>
  </action>
  <action name="openFile2Action">
   <property name="text">
    <string>Open cam-2 file</string>
   </property>
  </action>
  <action name="openFile3Action">
   <property name="text">
    <string>Open cam-3 file</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header>QVTKWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
